pkg/network/config/config.go

```diff
@@ -301,6 +308,8 @@ func New() *Config {
 		EnableHTTPSMonitoring: cfg.GetBool(join(netNS, "enable_https_monitoring")),
 		MaxHTTPStatsBuffered:  cfg.GetInt(join(smNS, "max_http_stats_buffered")),
 		MaxKafkaStatsBuffered: cfg.GetInt(join(smNS, "max_kafka_stats_buffered")),
+		UseGRPC:               cfg.GetBool(join(smNS, "use_grpc")),
+		GRPCSocketFilePath:    cfg.GetString(join(smNS, "grpc_socket_file_path")),
```

Comment:
To me, it makes more sense to put it under npm rather than usm
```suggestion
		UseGRPC:               cfg.GetBool(join(netNS, "use_grpc")),
		GRPCSocketFilePath:    cfg.GetString(join(netNS, "grpc_socket_file_path")),
```

Reply:
please add UTs